src/main/java/org/example/camunda/process/solution/facade/FormsController.java

```diff
@@ -36,20 +33,22 @@ public class FormsController extends AbstractController {
   public JsonNode getFormSchema(
       @PathVariable String processDefinitionId, @PathVariable String formKey)
       throws TaskListException, IOException {
+
     if (formKey.startsWith("camunda-forms:bpmn:")) {
       String formId = formKey.substring(formKey.lastIndexOf(":") + 1);
       String schema = taskListService.getForm(processDefinitionId, formId);
       return JsonUtils.toJsonNode(schema);
     }
-    Form form = formService.findByName(formKey);
+
+    Form form = formService.findFormJsonFileByFormKey(formKey);
```

Comment:
Why is this change needed?

Reply:
This change is probably, technically, not needed. But I think the code related to loading forms can be simplified and consolidated. 

I think it might be good to move any code related to loading forms into 2 methods, probably in the `FormService`: 

1. `findCamundaForm(processId, formId)`. This would read the form from bpmn files
2. `findJsonForm(formKey)`

Reply:
Hi @upgradingdave,

This means you would not use the tasklist api anymore, right ?